src/univers/version_range.py

```diff
@@ -868,6 +868,39 @@ def from_native(cls, string):
         return cls(constraints=constraints)
 
 
+class OpensslVersionRange(VersionRange):
+    """
+    Openssl version range.
+    openssl doesn't use <,>,<= or >=
+    For more see 'https://www.openssl.org/news/vulnerabilities.xml'
+
+    For exmaple::
```

Comment:
```suggestion
    For example::
```

Reply:
Yeah, I have implemented all those suggestions locally.. once the test for version comparison is ready I will push those changes